client/src/components/Public/Map/ClusterMarker/ClusterMarker.js

```diff
@@ -31,13 +32,14 @@ const ClusterMarker = ({
   zoomToLocation,
   onClick,
   openedAddressId,
+  useName,
 }: ClusterMarkerProps) => {
-  const MarkerText = <span className="marker__text">{cluster.numPoints}</span>
+  const MarkerText = <span className="marker__text">{useName ? cluster.id : cluster.numPoints}</span>
```

Comment:
I made it with labels, partly due to debbuging and partly because i liked it so, i can switch it to balloons but pinto said that it is ok this way. But i have no problems with baloons

Reply:
Ah, if it was approved by Pinto, it's ok. I see you've already changed this, so maybe keep it this way now and maybe later we can change it back if we feel like that. Sorry about confusion.